Refine layout and wording on page

phipsae opened the pull request:
## Description

Hi,
I've updated the layout and added more text to provide a clearer explanation of what we're doing (see the screenshots below).
I hope that's okay! 

Thanks @Pabl0cks it is already a nice page! Open for feedback :) 

<img width="1351" alt="Screenshot 2025-01-04 at 14 09 47" src="https://github.com/user-attachments/assets/07471cdb-f581-46f0-bc5e-83678e3ebe65" />

<img width="1325" alt="Screenshot 2025-01-04 at 14 09 56" src="https://github.com/user-attachments/assets/8eebfe0b-6d03-4439-84ac-e8d6bad5f39b" />

<img width="1368" alt="Screenshot 2025-01-04 at 14 10 01" src="https://github.com/user-attachments/assets/6fad5dff-8adb-494e-92ff-e3fbc1539f72" />

Pabl0cks commented on packages/nextjs/pages/batches/index.tsx:
```diff
@@ -172,22 +285,22 @@ const Batches = ({ batchData, openBatchNumber, openBatchStartDate }: PageProps)
       <div className="bg-[#EDEFFF] pt-16 lg:pt-24 pb-16">
         <div className="container max-w-[90%] lg:max-w-6xl mx-auto px-4 lg:px-12">
           {/* Next Batch CTA */}
```
The 'Next Batch CTA' is duplicated (mid-page and above table) - meant to have both?

Pabl0cks reviewed (commented):
Thanks a lot for the weaks! I like the new layout and content much more, just a few comments:

- "What are BuidlGuidl Batches?" sectopm could use a bit more spacing, at least in my laptop everything feels too tight. Mobile layout needs a review too.
- "What you'll learn?" section looks good to me!
- "How do Batches work?" something feels strange to me with this card layout, maybe that's a personal perception. I think we could use a bit extra spacing here too.

If needed we could ask Andrea, if she's available, to do a final review or give the designer touch.

phipsae commented:
Thanks for the fast feedback! :)

I fixed the mobile view (thanks for pointing out), added some more spacing and created two components. 
What do you think of creating more components to make the code more readable? 

For the "How do Batches work?" I wanted to have another card layout that doesn't repeat that from the "What are BuidlGuidl Batches?". For me it seems good, but open to change it :)

If Andrea has time that would be nice! :) 
Could you please ask her?

Pabl0cks commented:
> For the "How do Batches work?" I wanted to have another card layout that doesn't repeat that from the "What are BuidlGuidl Batches?". For me it seems good, but open to change it :)

Yeah I think is good enough! The pills felt a bit strange at start but it's fine! Everything looking good now, just a couple of nitpicks:

- Can you give more line spacing to the card titles here?

![image](https://github.com/user-attachments/assets/a3375873-c727-44d4-9882-2615dd989c61)


- What do you think about changing the order of the table and the CTA from the last section? Having already a CTA at top, I think visually it would look a bit better for the whole section. Ignore if you feel like it's better like it is 🙏

![image](https://github.com/user-attachments/assets/eb1b8b93-4684-4015-9abe-644cc8d577ae)

phipsae commented:
I added the spacing, definitely better! (Funny that I didn’t notice it earlier 😅)

As for the order of the CTA and the table, I tried moving the CTA to the bottom, but I didn’t like it much. I’m happy with how it is now, but if you want to change it, let me know!

Pabl0cks commented on packages/nextjs/pages/batches/index.tsx:
```diff
@@ -172,28 +267,7 @@ const Batches = ({ batchData, openBatchNumber, openBatchStartDate }: PageProps)
       <div className="bg-[#EDEFFF] pt-16 lg:pt-24 pb-16">
```
Maybe with `pt-2 lg:pt-8` is enough spacing?

```suggestion
      <div className="bg-[#EDEFFF] pt-2 lg:pt-8 pb-16">
```

phipsae replied:
Changed

Pabl0cks commented:
Just realized vercel build was failing, tried to `yarn build` in local and got this:

```
Build optimization failed: found pages without a React Component as default ex
port in
pages/batches/components/BatchCta
pages/batches/utils/formatDate
pages/batches/components/Card

See https://nextjs.org/docs/messages/page-without-valid-component for more info.
```

@phipsae think we need to move the new components to `/components` and the new util to `utils`

phipsae commented:
Thx! 
Fixed it. Now its successfully deployed. 
I gonna promote it to production :)
